apps/bfd-pipeline/bfd-pipeline-rif-load/src/main/java/gov/cms/bfd/pipeline/rif/load/RifLoader.java

```diff
@@ -79,6 +88,8 @@
    */
   private static final int RECORD_BATCH_SIZE = 100;
 
+  private static final int MAX_FILE_AGE = 40; // Days
```

Comment:
Add `DAYS` to the name of the variable so it's clear during usage.

Reply:
Good point. Will fix.